download_and_create_reference_datasets/v02/hail_scripts/write_exac.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+import hail as hl
+
+from hail_scripts.v02.utils.hail_utils import import_vcf
+
+CONFIG= {
+    '37': 'gs://seqr-reference-data/GRCh37/gnomad/ExAC.r1.sites.vep.vcf.bgz',
+    '38': 'gs://seqr-reference-data/GRCh38/gnomad/ExAC.r1.sites.liftover.b38.vcf.gz',
+}
+
+logger = logging.getLogger('v02.hail_scripts.create_exac_ht')
+
+
+def vcf_to_mt(path, genome_version):
+    '''
+    Converts exac vcf to mt and adds info.AF_POPMAX annotation.
+
+    :param path: vcf path
+    :param genome_version: genome version
+    :return:
+    '''
+    mt = import_vcf(path,
+                    genome_version=genome_version,
+                    min_partitions=1000)
+
+    # Annotate rows with popmax af if AC_POPMAX is not 'NA'. Value =
+    # AC/AN, and null otherwise.
+    cond = (mt.info.AC_POPMAX[mt.a_index-1] != "NA")
+    value = (hl.float((hl.int(mt.info.AC_POPMAX[mt.a_index - 1]) /
+                       hl.int(mt.info.AN_POPMAX[mt.a_index - 1]))))
+    alternative = hl.null(hl.tfloat)
+    mt = mt.annotate_rows(info=mt.info.annotate(AF_POPMAX=hl.cond(cond, value, alternative)))
```

Comment:
Would also discard unused fields as the 1st step 
https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/master/hail_scripts/v01/utils/add_exac.py#L101

Reply:
I didn't for this ref data for 2 reasons:
1. It does not conform to our convention of load and dump (e.g. load_from_vcf)
2. We might need those in the future.
The selection will happen when combining the data sets. I believe filtering at this level will reduce the size but not the performance time of combining the dataset, since selecting fields should be pretty fast.